src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  const {
+    user,
+    title,
+    completed,
+  } = todo;
+
+  return (
+    <article className={
+      completed
+        ? 'TodoInfo TodoInfo--completed'
+        : 'TodoInfo'
+    }
```

Comment:
You can use a classnames library here or look at the example: 
```suggestion
    <article className={`TodoInfo ${completed ? "TodoInfo--completed" : "" }`}
```